[CI] Solaris 11x64 rsyslog failed — rsyslog solaris11x64 rsyslog — BUILD FAILED: failed 'gmake -j2 ...' (failure)

## Solaris 11x64 rsyslog CI failure

**Summary:** BUILD FAILED: failed 'gmake -j2 ...' (failure)
**Builder:** rsyslog solaris11x64 rsyslog
**Build:** https://build.rsyslog.com/#/builders/77/builds/7154
**Revision:** 31a5887b75b81adf64eb0600c58ce7f7e45d96e1
**Worker:** slave-solaris11x64

### Steps

- 0: worker_preparation — success

- 1: set http proxy in git config — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/1/logs/stdio) 

- 2: git — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/2/logs/stdio) 

- 3: shell — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/3/logs/stdio) 

- 4: cleanup — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/4/logs/stdio) 

- 5: cleanup dependencies — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/5/logs/stdio) 

- 6: git branch information — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/6/logs/stdio) 

- 7: building librelp dependency — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/7/logs/stdio) 

- 8: building libfastjson dependency — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/8/logs/stdio) 

- 9: autoreconf for SunStudio Compiler — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/9/logs/stdio) 

- 10: shell_1 — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/10/logs/stdio) [config.log](https://build.rsyslog.com/#/builders/77/builds/7154/steps/10/logs/config_log) 

- 11: build with SunStudio — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/11/logs/stdio) 

- 12: shell_2 — failure — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/12/logs/stdio) [test-suite.log](https://build.rsyslog.com/#/builders/77/builds/7154/steps/12/logs/test-suite_log) 

- 13: gathering make check logs — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/13/logs/stdio) 

- 14: shell_3 — success — [stdio](https://build.rsyslog.com/#/builders/77/builds/7154/steps/14/logs/stdio)

--- a/tests/testsuites/ommail-smtp-server.py
+++ b/tests/testsuites/ommail-smtp-server.py
@@ -37,7 +37,7 @@ def main():
         srv.listen(1)
         port = srv.getsockname()[1]
         with open(args.port_file, "w", encoding="ascii") as fh:
-            fh.write(f"{port}\n")
+            fh.write("{}\n".format(port))
 
         srv.settimeout(30)
         conn, _addr = srv.accept()
@@ -62,7 +62,7 @@ def main():
                         data_lines.append(line)
                     continue
 
-                transcript.append(f"CMD:{line}")
+                transcript.append("CMD:{}".format(line))
                 command = line.split(" ", 1)[0].upper()
                 if command == "DATA":
                     in_data = True

